mix.exs

```diff
@@ -27,10 +28,12 @@ defmodule Airports.Mixfile do
 
   defp deps do
     [
-      {:nimble_csv, "~> 1.1"},
+      {:castore, "~> 1.0"},
```

Comment:
I don't see this package being used. Can it be removed?

Reply:
Yup, I think we can remove it. Will push here

Reply:
Seems we can't remove it as coveralls are using it for OTP 24 and below:
```
** (ExCoveralls.ReportUploadError) Failed to use any SSL certificates. We didn't find the :castore library,
and we couldn't use OS certificates because that requires OTP 25 or later.
If you want to use :castore, please add

  {:castore, "~> 1.0"}
```

Reply:
Ah check! Can you change it to `{:castore, "~> 1.0", only: :test},`? `excoveralls` seems only to be used at testing.